lib/Wikia/src/Service/User/Attributes/maintenance/cleanupMigratedAttributs.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Remove migrated attribute values from sharedDB. These are user attributes which have been migrated
+ * to the user attribute service. Any time we migrate additional attributes (eg, those found in 
+ * $wgPrivateUserAttributes), we can run this script again to clean up those attributes from the
+ * user_properties table in wikicities.
+ *
+ * Assumptions: The global variable $wgPublicUserAttributes is an array containing the names of
+ * all fully migrated attributes. Do NOT add any attributes to that array unless you have fully
+ * migrated all of its values from wikicities.user_properties to the attribute service.
+ */
+
+require_once( __DIR__ . '/../../../../../../../maintenance/Maintenance.php' );
+
+class CleanupMigratedAttributes extends Maintenance {
+
+    private $force;
+
+    public function __construct() {
+        parent::__construct();
+        $this->addOption( 'force', "Actually remove attributes from sharedDB. Script defaults to dry-run" );
+    }
+
+    public function execute() {
+        $this->force = $this->hasOption( 'force' );
+        $this->printAttrsToBeDeleted();
+        $this->deleteAttributes();
+        $this->printSuccess();
+    }
+
+    private function printAttrsToBeDeleted() {
+        global $wgPublicUserAttributes;
+        $this->output( "Preparing to delete following attributes from wikicities.user_properties:\n\n");
+        $this->output( implode( $wgPublicUserAttributes, "\n" ) );
+        $this->output( "\n\n" );
+        if ( $this->force ) {
+            $this->output( "NOT DRY RUN, ATTRIBUTES WILL BE DELETED\n" );
```

Comment:
I'd add info that deletion will begin in 5s so that the user can be more confident when interrupting the script.